testing/path.go

```diff
@@ -115,3 +115,97 @@ func (path *Path) RelayPacketWithResults(packet channeltypes.Packet) (*abci.Exec
 
 	return nil, nil, fmt.Errorf("packet commitment does not exist on either endpoint for provided packet")
 }
+
+// Setup constructs a TM client, connection, and channel on both chains provided. It will
+// fail if any error occurs. The clientID's, TestConnections, and TestChannels are returned
+// for both chains. The channels created are connected to the ibc-transfer application.
```

Comment:
```suggestion
// Setup constructs a TM client, connection, and channel on both chains provided. It will
// fail if any error occurs. The channels created are connected to the ibc-transfer application.
```